src/Tribe/Views/V2/Template/Page.php

```diff
@@ -157,8 +203,8 @@ public function maybe_hijack_main_query() {
 			return false;
 		}
 
-		// Store old posts
-		$wp_query->tribe_hijacked_posts = $wp_query->posts;
+		// Store old posts.
```

Comment:
The global `$wp_query` object might not be set; let's check the global `$wp_query` object is set and what we expect it to be and then move on, or bail.